index.bs

```diff
@@ -4829,8 +4829,7 @@ The below subsections comprise the current Web Authentication-specific security
 ## Cryptographic Challenges ## {#cryptographic-challenges}
 
 As a cryptographic protocol, Web Authentication is dependent upon randomized challenges
-to avoid replay attacks. Therefore, both {{PublicKeyCredentialCreationOptions/challenge}}'s
-and {{PublicKeyCredentialRequestOptions/challenge}}'s value MUST be randomly generated
+to avoid replay attacks. Therefore, the values of both {{PublicKeyCredentialCreationOptions}}.{{PublicKeyCredentialCreationOptions/challenge}} and {{PublicKeyCredentialRequestOptions}}.{{PublicKeyCredentialRequestOptions/challenge}} MUST be randomly generated
 by [=[RPS]=] in an environment they trust (e.g., on the server-side), and the
 returned challenge value in the client's
```

Comment:
Add a link to [`#dom-collectedclientdata-challenge`](https://w3c.github.io/webauthn/#dom-collectedclientdata-challenge) here?

Reply:
good suggestion, done.